pytest_celery/healthchecks/connection.py

```diff
@@ -21,6 +21,6 @@ def __call__(self) -> None:
 
         try:
             s.close()
-        except:
+        except ():
```

Comment:
Actually this means catch nothing (I think, I never tried). You can ignore this specific flake8 error.
In this case it would be beneficial to catch `socket.error`.
```suggestion
        except socket.error:
```